server/src/game/role_list.rs

```diff
@@ -231,8 +377,16 @@ impl RoleOutlineOptionRoles{
                 vec![*role]
         }
     }
-    pub fn is_subset(&self, other: &RoleOutlineOptionRoles) -> bool {
-        self.get_roles().iter().all(|r|other.get_roles().contains(r))
+    ///Returns true if this is a subset of rhs
+    pub fn subset_of(&self, rhs: &RoleOutlineOptionRoles) -> bool {
+        if self == rhs {return true};
+        let Self::RoleSet {role_set: rhs} = rhs else {return false};

```

Comment:
There isn't a role set with only one role, but if there was this would be an issue